lib/widgets/buildTasks.dart

```diff
@@ -61,138 +62,161 @@ class _TasksBuilderState extends State<TasksBuilder> {
+                            TextButton(
+                              onPressed: () {
+                                setStatus('deleted', task.uuid);
+                                DateTime? dtb = task.due;
+                                dtb = dtb!.add(Duration(minutes: 1));
+                                final FlutterLocalNotificationsPlugin
+                                _flutterLocalNotificationsPlugin =
+                                FlutterLocalNotificationsPlugin();
+                                _flutterLocalNotificationsPlugin.cancel(
+                                    dtb.day * 100 +
+                                        dtb.hour * 10 +
+                                        dtb.minute);
+
+                                print("Task due is" + dtb.toString());
+                                print(dtb.day * 100 +
+                                    dtb.hour * 10 +
+                                    dtb.minute);
+                                Navigator.of(context).pop();
+                              },
+                              child: const Text('Yes'),
+                            ),
+                            TextButton(
+                              onPressed: () {
+                                Navigator.of(context).pop();
+                              },
+                              child: const Text('No'),
+                            ),
+                            TextButton(
+                              onPressed: () {
+                                Navigator.of(context).pop();
+                              },
+                              child: const Text('Cancel'),
+                            ),
+                          ],
+                        );
+                      },
+                    );
+                  },
+                  icon: Icons.delete,
+                  label: "DELETE",
+                  backgroundColor: Colors.red,
+                ),
+              ],
+            ),
+            child: Card(
+              color:
+              AppSettings.isDarkMode ? Palette.kToDark : Colors.white,
+              child: InkWell(
+                splashColor: AppSettings.isDarkMode
+                    ? Colors.black
+                    : Colors.grey.shade200,
+                onTap: () => Navigator.push(
+                  context,
+                  MaterialPageRoute(
+                    builder: (context) => DetailRoute(task.uuid),
                   ),
-                )
+                ),
+                child: TaskListItem(
+                  task,
+                  pendingFilter: widget.pendingFilter,
+                  darkmode: AppSettings.isDarkMode,
+                ),
+              ),
+            ),
+          )
               : Card(
-                  color:
-                      AppSettings.isDarkMode ? Palette.kToDark : Colors.white,
-                  child: InkWell(
-                    customBorder: RoundedRectangleBorder(
```

Comment:
same here

Reply:
@Interstellar07 @NishantSinghal19 please check

Reply:
will be resolved if holds true, Thanks!